schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+                "Grant",
+                "Instrument",
+                "JournalArticle",
+                "JournalIssue",
+                "JournalVolume",
+                "Journal",
+                "PeerReview",
+                "PhysicalObject",
+                "ProceedingsArticle",
+                "ProceedingsSeries",
+                "Proceedings",
+                "ReportComponent",
+                "ReportSeries",
+                "Report",
+                "Software",
+                "Other"
+            ]
+        },
+        "additional_type": {
+            "description": "The additional type of the resource.",
+            "type": "string"
+        },
+        "url": {
+            "description": "The URL of the resource.",
+            "type": "string",
+            "format": "uri"
+        },
+        "creators": {
+            "description": "The creators of the resource.",
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "description": "The unique identifier for the creator.",
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "type": {
+                        "description": "The type of the creator.",
+                        "type": "string",
+                        "enum": [
+                            "Organization",
+                            "Person"
+                        ]
+                    },
+                    "name": {
+                        "description": "The name of the creator.",
+                        "type": "string"
+                    },
+                    "givenName": {
+                        "description": "The given name of the creator.",
+                        "type": "string"
+                    },
+                    "familyName": {
+                        "description": "The family name of the creator.",
+                        "type": "string"
+                    },
+                    "affiliation": {
+                        "$ref": "#/definitions/affiliations"
+                    }
+                }
+            },
+            "minItems": 1,
+            "uniqueItems": true
+        },
+        "publisher": {
+            "description": "The publisher of the resource.",
+            "type": "object",
+            "properties": {
+                "id": {
+                    "description": "The identifier for the publisher.",
+                    "type": "string"
+                },
+                "name": {
+                    "description": "The name of the publisher.",
+                    "type": "string"
+                }
+            },
+            "required": [
+                "name"
+            ]
+        },
+        "date": {
+            "description": "The dates for the resource.",
+            "type": "object",
+            "format": "date"
```

Comment:
what event does this date refer to?

Reply:
I changed the date list in DataCite metadata to a date dict in commonmeta. I found that easier to work with, in particular conversion from/to other formats.

Reply:
have you got some examples of data for this field? `date` isn't defined in `$defs` so I was wondering how you're tying dates to various events in the "life cycle" of a DOI'd object.

Reply:
Thanks for spotting this. I had used date types mostly aligned with the DataCite `dateType` (but using `published` instead of `issued`), but hadn't documented it in the schema.

I updated the schema to `0.10.2` with the various date types that I have used so far, not yet implemented in the Python library: https://github.com/front-matter/commonmeta-py/blob/main/commonmeta/resources/commonmeta_v0.10.2.json